web/client/translations/data.it-IT.json

```
@@ -2848,8 +2850,9 @@
         "cacheT": "Pulisci la cache",
         "save": "Salva regola corrente",
         "close": "Esci da modifica regola",
-        "showAllRules": "Mostra tutte le regole applicabili",
-        "filterRuleList": "Filtra la lista usando il valore selezionato"
+        "showAllRules": "Mostra solo le regole che corrispondono specificamente in base al valore di questo campo. Deseleziona per includere anche le regole dove questo campo non è considerato.",
+        "filterRuleList": "Mostra tutte le regole applicate, anche se non corrispondono specificamente in base a questo campo. Seleziona per visualizzare solo le regole che si applicano a causa di questo valore.",
```

```suggestion
        "filterRuleList": "Mostra solo le regole dove il valore specificato non è presente. Deseleziona per includere solo le regole che corrispondono strettamente al valore indicato per questo campo.",
```

---

This is incorrect for 2 reasons: 
- The tooltip is shown when the item is unchecked, so the user have to (check), not "uncheck" (deseleziona)
- In the condition where the tooltip is shown, the table show all rules that match, including both request that match and that are empty. Here your text says to show "only" unselected.
```suggestion
        "filterRuleList": "Attualmente la tabella mostra tutte le regole applicate per questa condizione, anche quelle per cui questo valore non è impostato . Seleziona per visualizzare solo le regole che si applicano specificatamente per questo valore.",
```